src/Microsoft.Identity.Client/AppConfig/ConfidentialClientApplicationBuilder.cs

```diff
@@ -85,9 +97,9 @@ public ConfidentialClientApplicationBuilder WithUserTokenCache(TokenCache tokenC
         }
 
         /// <summary>
-        /// 
+        /// Sets the certificate associated with the application
         /// </summary>
-        /// <param name="certificate"></param>
+        /// <param name="certificate">The X509 certificate used as credentials to prove the identity of the application to Azure AD.</param>
         /// <returns></returns>
         public ConfidentialClientApplicationBuilder WithX509Certificate2(X509Certificate2 certificate)
```

Comment:
There was a request from @jeferrie@microsoft.com  in a previous PR about changing this to be WithCertificate(X509Certifiate2 certificate) and possibly having other overloads if we add different certificate types.  thoughts?